opsdroid/core.py

```diff
@@ -286,12 +286,12 @@ def setup_skills(self, skills):
 
         """
         if "parsers" in self.config:
-            parsers = self.config["parsers"] or []
-            rasanlu = [p for p in parsers if p["name"] == "rasanlu"]
-            if len(rasanlu) == 1 and (
-                "enabled" not in rasanlu[0] or rasanlu[0]["enabled"] is not False
+            parsers = self.config["parsers"] or {}
+            rasanlu = parsers.get("rasanlu")
+            if rasanlu and (
+                "enabled" not in rasanlu or rasanlu["enabled"] is not False
```

Comment:
We could make `enabled` a default key in the loader because this pattern is repeated a lot and is a bit gross.

Would be nicer if we could just have

```
            if rasanlu and rasanlu["enabled"]:
```